### ISSUE TITLE
Student's performance on resnet8

### ISSUE THREAD

**Issue by aminshabani** [imirzadeh/Teacher-Assistant-Knowledge-Distillation#3]
The result which I got from running the code for resnet8 as the student is much lower than the result that is mentioned in the paper, I think one reason could be due to the "crop=False" which removes RandomHorizontalFlip from augmentation.
Is there any other reason or did I something wrong?

**Comment by imirzadeh** [comment]
Hi,   

I can't comment on your code since I don't know it. But there are some tricks to train resnets like learning rate scheduling or using augmentations like cropping/rotating and ... which will increase the performance. Also, because NNs are sensitive to initialization, you might need to run the code with different seeds.   

Since this issue is not related to my code, I will close it for now. Feel free to open another issue if you couldn't get the reported results in the paper by running this code.
